dynamic/Dockerfile

```diff
@@ -116,6 +116,17 @@ RUN [[ "${TARGETPLATFORM}" != 'linux/ppc64le' ]] || ( \
         && rm -rf /tmp/* \
     )
 
+RUN export ARCH=${TARGETPLATFORM////-} \
```

Comment:
Looks good so far, except for the `ARCH` that needs to be handled.

Reply:
Can you please elaborate on the requested change?